spec/helpers/etablissement_helper_spec.rb

```diff
@@ -1,31 +1,42 @@
 RSpec.describe EtablissementHelper, type: :helper do
   let(:code_effectif) { '00' }
   let(:raison_sociale) { 'GRTGaz' }
+  let(:enseigne) { "mon enseigne" }
   let(:nom) { 'mon nom' }
   let(:prenom) { 'mon prenom' }
-  let(:entreprise_params) do
+  let(:etablissement_params) do
     {
+      enseigne: enseigne,
       entreprise_capital_social: 123_000,
       entreprise_code_effectif_entreprise: code_effectif,
       entreprise_raison_sociale: raison_sociale,
       entreprise_nom: nom,
       entreprise_prenom: prenom
     }
   end
-  let(:etablissement) { create(:etablissement, entreprise_params) }
+  let(:etablissement) { create(:etablissement, etablissement_params) }
 
   describe '#raison_sociale_or_name' do
     subject { raison_sociale_or_name(etablissement) }
 
+    context 'when enseigne exist' do
```

Comment:
Tu aurais moyen d'élaborer, pour que ça soit clair dans quel cas on a une enseigne ?

Ça serait un truc du genre "when the etablissement is not the siege" ?